INN/umbrella-currentorg
# Hide fee plan images on submission received page

## Changes

This pull request makes the following changes:

- Adds `display-none-important` class to the fee level images, but uses the `'wpbdp_before_submit_listing_page` hook to add css to display them on the submit listing page.

## Why



For #105 

## Testing/Questions

Features that this PR affects:

- Submit listing page
- Submission received page

Steps to test this PR:

1. Go to submit a listing page
2. Verify images show
3. Submit listing and go to submission received page
4. Verify images do not show
